packages/core/stories/components/Header/Header.stories.js

```diff
@@ -91,240 +94,16 @@ const headerUtilSearch = (
-            },
-            {
-              href: '#',
-              text: 'Judicial Branch'
-            },
-            {
-              href: '#',
-              text: 'Legislative Branch'
-            },
-            {
-              href: '#',
-              text: 'Local Government'
-            }
-          ],
-          text: 'Your Government'
-        }
-      ],
-      onNavigateCallBack: function noRefCheck() {}
-    }}
-    searchRedirect={{
-      baseUrl: 'https://search.mass.gov',
-      queryParams: {
-        page: '1'
-      },
-      searchTermParam: 'q'
-    }}
-    siteLogo={{
-      image: {
-        alt: 'Massachusetts state seal',
-        height: 45,
-        src: logo,
-        width: 45
-      },
-      siteName: 'Mass.gov',
-      title: 'Mass.gov homepage',
-      url: {
-        domain: 'https://www.mass.gov/'
-      }
-    }}
-    utilityNav={{
-      items: [
-        {
-          ariaLabelText: '',
-          closeText: undefined,
-          icon: 'IconBuilding',
-          panel: undefined,
-          text: 'All Budgets'
-        },
-        {
-          ariaLabelText: '',
-          closeText: 'Close',
-          icon: 'building',
-          panel: {
-            description: {
-              text: 'The <a href="#">A-Z Organizations page</a> provides an alphabetical listing of government organizations, including commissions, departments, and bureaus.'
-            }
-          },
-          text: 'State Organizations'
-        },
-        {
-          ariaLabelText: 'Log in to the most requested services',
-          closeText: 'Close',
-          icon: 'IconLogin',
-          panel: {
-            description: {
-              text: 'These are the top requested sites you can log in to access state provided services'
-            },
-            links: [
-              {
-                href: 'https://uionline.detma.org/Claimant/Core/Login.ASPX',
-                text: 'Unemployment Online',
-                type: 'external'
-              },
-              {
-                href: 'https://sso.hhs.state.ma.us/oam/server/obrareq.cgi?encquery%3DA2%2Fmo5AkZreDycpyP0JZAEOYGvW2hviyNhH9Sht2xPp0V1%2BBtWfHnmRGr6zNHOqOlcjphPk7p6bpHHRyNzzk9IYQ%2FcN%2B%2FIcqL2ThnI217OsIKZepptTpGBx83SI0NWjsE7vDi72caItXWlelbGQT7ePanlrVUUy2%2Fj1UEUaXi5G7m47KO9djBnoetZRCtp9G2ZTNFf6zvCGU7Cs02AXYUj2JMH4aqol%2Bh3OK6uhJNNkFvwQ1MFRUa4gR1az4iaW9u83ExKb2a9eDv8ZIUqhlq3%2BNVGTqZHAsHX4KOONSGQRBwCtLNPWwruacjdd9CaEqeIJ2tnP45KrM93edZ6zU1yoWGbAp%2BUWWMqk4HyrtuA8%3D%20agentid%3Dwebgate1%20ver%3D1%20crmethod%3D2',
-                text: 'Virtual Gateway (SNAP)',
-                type: 'external'
-              },
-              {
-                href: 'https://ecse.cse.state.ma.us/ECSE/Login/login.asp',
-                text: 'Child Support Enforcement',
-                type: 'external'
-              }
-            ]
-          },
-          text: 'Log in to...'
-        }
-      ]
-    }}
+    mainItems={mainNavData}
+    utilityItems={[]}
```

Comment:
Set UtilityNav to null here:
```suggestion
    UtilityNav={null}
```

Reply:
won't setting it to null remove the utility nav items? I'd like to show the utility nav example same what's live on mass.gov

Reply:
Oh! No you're right, disregard.